src/v/pandaproxy/error.cc

```diff
@@ -246,6 +246,7 @@ std::error_condition make_error_condition(std::error_code ec) {
         case kec::rebalance_in_progress:
         case kec::throttling_quota_exceeded:
         case kec::reassignment_in_progress:
+        case kec::no_reassignment_in_progress:
```

Comment:
@BenPope opinion on this being retriable or not?

Reply:
Oh I see what you're saying. Let me change where I put this error.